local-setup/README.md

```diff
@@ -0,0 +1,36 @@
+#  How to use the local-setup
+
+## Prerequisite
+- worker built with `make`
+- integritee-node built with `cargo build --release`
+
+## Steps
+Adapt or create your own config file, as in the example of `simple-config.json`. Be mindful of the ports in case you're running the script on a server multiple people are working on.
+
+### Launch worker and node in terminal one
+You can launch the workers and the node with:
+```bash
+cd <worker directory>
```

Comment:
I don't think this is needed. It is obvious from the command below.

Reply:
removed